common/config/rush/pnpm-lock.yaml

```diff
@@ -69,41 +69,41 @@ importers:
       webpack: 4.42.0
     dependencies:
       '@bentley/icons-generic-webfont': 1.0.34
```

Comment:
why was lockfile updated?

Reply:
> Next time could just do `rush update --full` to update the lockfile.

Wanted to update as little as possible because if dependencies introduced a new bug, it would have been hard to find the cause after a massive dep update.

Also, [`rush audit` output](https://github.com/iTwin/viewer/actions/runs/3828265461/jobs/6513659549#step:4:763) before updating lockfile. Most errors were caused by outdated versions in lockfile.